### PR TITLE
update user profile

### PR DESCRIPTION
**Pull request by ducls-2223**
ready

### CODE REVIEW

**Comment by ducls-2223** [comment]
ready

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    private $dataUpdate = [];
+
+    public function __construct(UserRepositoryInterface $userRepository)
+    {
+        $this->userRepository = $userRepository;
+    }
+
+    public function show($id)
+    {
+        if($id != auth()->id()) {
```
Comment (huyenminnn) [inline app/Http/Controllers/UserController.php]: convention

File: app/Repositories/User/UserRepository.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace App\Repositories\User;
+
+use App\Repositories\BaseRepositories;
+
+class UserRepository extends BaseRepositories implements UserRepositoryInterface
+{
+    public function getModel()
+    {
+        return \App\Models\User::class;
```
Comment (huyenminnn) [inline app/Repositories/User/UserRepository.php:11]: User::class

File: resources/views/user/profile.blade.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            <div class="col-sm-8 offset-2">
+                <form>
+                    @csrf
+                    <div class="form-group">
+                        <label for="image">{{ __('user.avatar') }}</label>
+                        @if (!auth()->user()->image)
+                            <img src="{{ asset('storage/images/member-default.jpg') }}" width="80" height="80" alt=""
+                                 class="rounded-circle avatar-user">
+                        @else
+                            <img src="{{ asset('storage/images/' . $user->image) }}" width="80" height="80" alt=""
```
Comment (huyenminnn) [inline resources/views/user/profile.blade.php]: css

File: resources/views/user/profile.blade.php [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                               placeholder="{{ __('user.enter_new_password') }}">
+                    </div>
+                    <div class="form-group">
+                        <label for="confirm-password">{{ __('user.confirm_password') }}</label>
+                        <input type="password" class="form-control" id="confirm-password" name="password_confirmation"
+                               placeholder="{{ __('user.enter_confirm_password') }}">
+                    </div>
+                    <button type="submit" class="btn btn-success" id="user-btn-save" value="{{ auth()->id() }}"><i class="fa fa-floppy-o pr-2"
+                                                                     aria-hidden="true"></i>{{ __('user.save') }}
+                    </button>
```
Comment (huyenminnn) [inline resources/views/user/profile.blade.php]: indent

**Comment by ducls-2223** [comment]
ready

**Pull request merged**